test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3499,5 +3499,63 @@ public void testScoreboardHTMLLocations() throws Exception {
         assertEquals("Public HTML Directory", "public_html_dir", publicDir);
 
     }
+    
+    
+    public void testTenProbsisStopOnFirstFailedTestCase() throws Exception {
+        
+        String sampleName = "tenprobs";
+        IInternalContest contest  = fullLoadSampleContest(sampleName);
+        assertNotNull(contest);
+
+        Problem[] problems = contest.getProblems();
+        assertEquals("Num problems ",10, problems.length);
+        
+        for (Problem problem : problems) {
+            assertTrue(problem.getShortName()+" stop on first ", problem.isStopOnFirstFailedTestCase());
+        }
+        
+    }
+    
+    public void tesproblemflagtestStopOnFirstFailedTestCase() throws Exception {
```

Comment:
If I recall correctly, JUnit methods must start with the characters "test...".   This method doesn't do that, so I don't think it will be recognized by JUnit as a Test.   Any anyway, it's a weird name for a method...

Reply:
pushed fix.  renamed method.  (when renamed, executed and failed, now passes)